controllers/project_update_test.go

```diff
@@ -97,24 +110,13 @@ func tProjectUpdateExistent(db *gorm.DB, testUsername string, testJwt string, t
 		ReadmeLink:     "http://NewRepoLink/README",
 	}
 
-	// Test with invalid new secondary mentor
```

Comment:
One last thing. Can you add a test that checks if giving an empty string unsets the mentor?

Reply:
Let's do this at a later point.